<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomTextGenerator</class>
 <widget class="QDialog" name="RandomTextGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Text Generator</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="5" column="0" colspan="3">
       <widget class="QRadioButton" name="wordOrder1RadioButton">
        <property name="text">
         <string>First order word</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QRadioButton" name="wordOrder2RadioButton">
        <property name="text">
         <string>Second Order Words</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QRadioButton" name="wordOrder3RadioButton">
        <property name="text">
         <string>Third Order Words</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="18" column="0" colspan="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Length:</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="22" column="0" colspan="3">
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="23" column="0" colspan="3">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="19" column="2">
       <widget class="QSpinBox" name="lengthSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="wordOrderXRadioButton">
        <property name="text">
         <string>Higher Order:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSpinBox" name="wordOrderSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="characterOrder1RadioButton">
        <property name="text">
         <string>First Order Characters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="characterOrder2RadioButton">
        <property name="text">
         <string>Second Order Characters</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QRadioButton" name="characterOrder3RadioButton">
        <property name="text">
         <string>Third Order Characters</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="characterOrderXRadioButton">
        <property name="text">
         <string>Higher Order:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="characterOrderSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="0" colspan="3">
       <widget class="QCheckBox" name="upperCaseCheckBox">
        <property name="text">
         <string>ignore case</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QCheckBox" name="punctationCheckBox">
        <property name="text">
         <string>punctation</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="3">
       <widget class="QCheckBox" name="latexInput">
        <property name="text">
         <string>LaTeX input</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QCheckBox" name="exportCheckBox">
        <property name="text">
         <string>Save to:</string>
        </property>
       </widget>
      </item>
      <item row="17" column="2">
       <widget class="QLineEdit" name="exportFileNameLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="outputEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RandomTextGenerator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
